packages/context/src/__tests__/acceptance/class-level-bindings.acceptance.ts

```diff
@@ -51,7 +51,11 @@ describe('Context bindings - Injecting dependencies of classes', () => {
         throw new Error('ctx.get() should have failed');
       },
       function onError(err) {
-        expect(err).to.match(/resolve.*InfoController.*argument 1/);
+        expect(err.message).to.eql(
+          'Cannot resolve injected arguments for InfoController.[0]: ' +
+            'The arguments[0] is not decorated for dependency injection, ' +
+            'but a value is not supplied',
```

Comment:
I find this error message a bit difficult to comprehend. How about this?

```
The arguments[0] is not decorated for dependency injection and the caller did not supply any value.
```

The parts "the arguments[0]" and "InfoController.[0]" look a bit weird too, can we make it easier to understand too? Assuming they are both referring to the first constructor argument:

```
Cannot resolve injected arguments for InfoController constructor:
The first argument is not decorated for dependency injection
and the caller did not supply any value for this argument.
```

(Positional numbers to use: `first`, `second`, `third`, `4th`, `5th` etc. )

Reply:
I'll defer this fix to another PR as the latest version of my PR does not impact this code.